src/main/java/com/kau/capstone/global/common/Gender.java

```diff
@@ -0,0 +1,21 @@
+package com.kau.capstone.global.common;
+
+import jdk.jfr.ContentType;
+
+public enum Gender {
+    MALE(1), FEMALE(2);
+    private final Integer value;
+
+    Gender(int value) {this.value = value;}
```

Comment:
P1: 이거는 아래처럼 정리부탁드립니다. `PetType`도 반영 부탁드려요

```suggestion
    MALE(1), FEMALE(2);
    
    private final Integer value;

    Gender(int value) {
        this.value = value;
    }
```

Reply:
1번으로 정리해두겠습니다~